about.html

```diff
@@ -0,0 +1,108 @@
+      <section class="fade-image">
+        <span class="img"></span>
+    </section>
+    <section class="rest-aboutpage">
+      <section class="about-text-section">
+        <h2 class="about-text-2">About Us</h2>
+        <article class="outline">
+        <p class="about-paragraph-home-1">  We are a grobal company that fights for the rights for those who cannot fight for themselves.<br><br>We would love to here from you and what we can do for you.Advocating for positive change with people over 80 countries and growing!</p>
+        </article>
+        <p class="p2">Please contact us or email us at this address if you want any form of help or questions about our mission and countires we operate from.<br><span> justiceforall@gmail.com</span></p>
+      </section>
+      <section class="mission">
+        <article class="blackheader">
+          <h2 class="main-text">Our Mission</h2>
+          <img src="assets/images/patterns/underline.png" alt="underline for text">
+        </article>
+        <p class="missionstatement">Our mission is to give every one a voice in the world and have every right to resources,education,gorvenment and civilisation among other things</p>
+        <img src="assets/images/mission.png" class="imagmission" alt="people s arms image">
+      </section>
+      <section class="past">
+        <article class="blackheader">
+          <h2 class="main-text">Past Trials</h2>
+          <img src="assets/images/patterns/underline.png" alt="underline for text">
+        </article>
+        <p class="missionstatement">take a look at some of our past trials and how we handled them with leading experts in law.</p>
+
+        <section class="past-images">
+          <figure class="img1"><img src="assets/images/haiti.jpg"  class="img-deep" alt="haiti images"></figure>
+          <figure class="img"><img src="assets/images/ukraine.jpg" class="img-deep" alt="haiti images"></figure>
+        </section>
+      </section>
+    </section>
+    </main>
+    <footer class="container-fluid container-default">
+      <section class="blacksection">
+       <article class="blackheader">
+         <h2 class="main-text">Clients</h2>
+         <img src="assets/images/patterns/underline.png" alt="underline for text">
+       </article>
+       <section class="companies">
+         <img src="assets/images/mozilla.png" alt="a company we have worked with">
+         <img src="assets/images/google.png" alt="a company we have worked with">
+         <img src="assets/images/naver.png" alt="a company we have worked with">
+         <img src="assets/images/microsoft.png" alt="a company we have worked with">
+         <img src="assets/images/airbnb.png" alt="a company we have worked with">
+       </section>
+      </section>
+      <section class="whitesection">
+        <section class="comImg">
+          <img src="assets/images/creativecommons.png" alt="a logo for copyright">
+          <h2 class="copyrightheader">Creative<br>Commons Grobal</h2>
```

Comment:
- [ ] please your desktop home page is missing a the partners section and footer, it includes a logo to the left and some text to the right. please kindly add that like in the original design.
original design
![Screenshot (1007)](https://user-images.githubusercontent.com/65298048/234981500-d98ec3ad-fddb-4973-ad4e-1dc9b0b17724.png)